app.js

```diff
@@ -17,10 +18,11 @@ db.on('error', console.error.bind(console, 'MongoDB connection error:'))
 
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: false }))
-app.use('/players', player)
+app.use(morgan('combined'))
+app.use('/', player)
 
-let port = 80
+let port = process.env.PORT || 80
```

Comment:
use `const`, i would change it to be actually `|| 8000` and set the dockerfile like the other examples to set `PORT=80`